.evergreen/run-tests.sh

```diff
@@ -2,28 +2,25 @@
 set -o xtrace   # Write all commands first to stderr
 set -o errexit  # Exit the script with error if any of the commands fail
 
-# Supported/used environment variables:
-#       AUTH                    Set to enable authentication. Defaults to "noauth"
-#       SSL                     Set to enable SSL. Defaults to "nossl"
-#       MONGODB_URI             Set the suggested connection MONGODB_URI (including credentials and topology info)
-#       MARCH                   Machine Architecture. Defaults to lowercase uname -m
-
-AUTH=${AUTH:-noauth}
-SSL=${SSL:-nossl}
-MONGODB_URI=${MONGODB_URI:-}
-
+# variables
+PROJECT_DIRECTORY=${PROJECT_DIRECTORY:-$PWD}
+MONGODB_URI=${MONGODB_URI:-"NO_URI_PROVIDED"}
+SWIFT_VERSION=${SWIFT_VERSION:-4.2}
+INSTALL_DIR="${PROJECT_DIRECTORY}/opt"
+TOPOLOGY=${TOPOLOGY:-single}
 OS=$(uname -s | tr '[:upper:]' '[:lower:]')
-[ -z "$MARCH" ] && MARCH=$(uname -m | tr '[:upper:]' '[:lower:]')
 
-if [ "$AUTH" != "noauth" ]; then
-  export MONGOC_TEST_USER="bob"
-  export MONGOC_TEST_PASSWORD="pwd123"
-fi
+# enable swiftenv
+export SWIFTENV_ROOT="${INSTALL_DIR}/swiftenv"
+export PATH="${SWIFTENV_ROOT}/bin:$PATH"
+eval "$(swiftenv init -)"
+
+# switch swift version, and run tests
+export PKG_CONFIG_PATH="${INSTALL_DIR}/lib/pkgconfig"
 
-if [ "$SSL" != "nossl" ]; then
-   export MONGOC_TEST_SSL_PEM_FILE="$DRIVERS_TOOLS/.evergreen/x509gen/client.pem"
-   export MONGOC_TEST_SSL_CA_FILE="$DRIVERS_TOOLS/.evergreen/x509gen/ca.pem"
-fi
+# override where we look for libmongoc
+export LD_LIBRARY_PATH="${INSTALL_DIR}/lib"
+export DYLD_LIBRARY_PATH="${INSTALL_DIR}/lib"
 
-echo "Running $AUTH tests over $SSL, connecting to $MONGODB_URI"
-make test
+swiftenv local $SWIFT_VERSION
+MONGODB_TOPOLOGY=${TOPOLOGY} MONGODB_URI=$MONGODB_URI make test
```

Comment:
So when looking at the evg output you posted in slack, it's pretty hard to find the log output for the tests, and it's not at all possible in the evg ui to navigate at a per-test level. Should it be part of future work to prettify this output, or is it worth doing now as part of this PR?

e.g. in the tools we have:
![image](https://user-images.githubusercontent.com/936020/56316411-335f9900-6128-11e9-8b1b-fd0a98618963.png)

Reply:
I think we should absolutely do this work, but in the future. This work today was meant to prove we could run on these configurations, but there exists an outstanding good deal of work to make our evergreen lives much better.